sdk/identity/azure-identity/pom.xml

```diff
@@ -88,7 +88,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.5.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
```

Comment:
Same as above. Use unreleased dependency.

Reply:
I think we'll need to use beta_  dependency right ?

Reply:
Fixed.

Reply:
I have zero clue. the comment in version_client.txt says:
```
# Released Beta dependencies: Copy the entry from above, prepend "beta_", remove the current
# version and set the version to the released beta. Released beta dependencies are only valid
# for dependency versions. These entries are specifically for when we've released a beta for
# a library but need to keep the dependency version at the latest released GA.
# Format;
# beta_<groupId>:<artifactId>;dependency-version
# note: Released beta versions will not be manipulated with the automatic PR creation code.
```
But this is not released yet.

Reply:
Yeah we can update this, once its released.

Reply:
@jianghaolu and I chatted, it'll be unreleased_ (aka using the source built binary) until it is released. In a nutshell unreleased_ is the way to use the "current" source version of a library outside of that library's pipeline. Beta_ is the way to use a version that's already been released (regardless of where from). A beta_ tag should not get manipulated in the From Source run, it should always use that version.

Reply:
Yeah, in master branch.